common/pango.c

```diff
@@ -10,8 +10,9 @@
 #include "log.h"
 #include "stringop.h"
 
+#define MAX_CHARS 16384
```

Comment:
The stack size on Alpine in 80K. We should be careful not to allocate too-large objects on the stack. 16K seems kind of okay considering there are not nested calls to these functions, but anything larger would be too large.

Reply:
16K does seem excessive. In commit cab13 it was just 2K.

Reply:
It has been increased because some strings were truncated. We may just want to use dynamic allocation depending on the length of the string in this case.

Reply:
I'll address this in another PR as long as that's alright with you.

Reply:
Fine by me